ui/src/app/views/cohort-edit/component.html

```diff
@@ -1,18 +1,60 @@
-<h2>Set Cohort Info</h2>
-<div class="form-area">
-  <div>
-    <label>Cohort Name:</label>
-    <input type="text" class="input name" name="cohort-name" autofocus
-        [(ngModel)]="cohort.name">
-  </div>
-  <div>
-    <label>Description:</label>
-    <textarea class="input description" name="cohort-description"
-        [(ngModel)]="cohort.description">
-    </textarea>
-  </div>
-  <div class="button">
-    <button class="btn save-button" [clrLoading]="loading" (click)="save()">Save Cohort</button>
-    <button class="btn" (click)="cancel()">Cancel</button>
+<div class="row">
```

Comment:
Is this new code? Is it related to this change?

Reply:
Wrapping the form controls in a form and a grid is new, yes.  I guess they're not strictly necessary although using form controls with `FormControl`s that aren't wrapped in a `form` tag requires a slightly different technique.